mvrx-mocking/src/main/kotlin/com/airbnb/mvrx/mocking/MockBuilder.kt

```diff
@@ -1460,18 +1471,20 @@ open class MavericksViewMocks<V : MockableMavericksView, Args : Parcelable> @Pub
 
             numAllowedCreationsOfMocks.incrementAndGet()
 
-            val mocks =
-                view.call<MavericksViewMocks<out MockableMavericksView, out Parcelable>>("provideMocks")
+            val mocks= mavericksViewMocks(view)
```

Comment:
needs a space. You can run ./gradlew detekt to apply detekt an ktlint checks locally